source/psa-config.rst

```diff
@@ -0,0 +1,178 @@
+  #
+  #       0 Disable
+  #       1 Persist only write settings
+  #       2 Persist both Read and Write settings
+
+  persistPrimarySettings=0
+
+  # == proxysql ===================================================
+  # The proxysql section is for ProxySQL-specific information.
+  #
+  # These settings will be read and used whenever the scheduler is run.
+  #
+  [proxysql]
+  port = 6032
+  host = "127.0.0.1"
+  user = "<valid user to connect from real ip as for proxysql_server table>"
+  password = "<password>"
+  clustered = false
+  lockfilepath ="/var/run/pxc_scheduler_handler"
+  respectManualOfflineSoft=false
+
+
+  #== global ======================================================
+  # The global section are for variables that are not ProxySQL or
+  # cluster specific.
+  #
+  # These settings will be read and used whenever the scheduler is run.
+  #
+  [global]
+  debug = true
+
+  # stdout: output is redirects output to proxysql logs
+  # file: output is written to the file pointed by logFile
+  logTarget = "file" #stdout | file
+
+  # Defines the log level to be used
+  # Allowed options are [error,warning,info,debug]
+  logLevel = "info"
+  logFile = "/var/log/pxc_scheduler_handler/pscheduler.log"
+
+  daemonize = false
+  daemonInterval = 2000
+
+  # boolean variable which enables reporting of statistics
+  performance = true
+
+  # Not used currently
+  OS = "na"
+
+  # Time in seconds after which the file lock is considered expired [local
+  instance lock]
+    lockfiletimeout = 60 #seconds
+
+  # Time in seconds after which the cluster lock is considered expired
+  lockclustertimeout = 600 #120 # seconds
+
+
+  #== setup =======================================================
+  # These variables are used only upon Setup
+  # Changing these variables after setup will not affect operation
+  #
+  [setup]
+
+  # --------------------------------
+  # The clusterAppUser is the ProxySQL user account that should be
+  # used by clients to access the cluster.
+  #
+  # Uncomment the following options (clusterAppUser and clusterAppUserPassword)
+  # to enable the setting of the clusterAppUser for this cluster.
+  #
+  #clusterAppUser="proxysql_user"
+  #clusterAppUserPassword="passw0rd"
+
+  # --------------------------------
+  # The monitorUser is used by ProxySQL to access the servers and
+  # check the connections.
+  #
+  monitorUser="monitor"
+  monitorUserPassword="monitor"
+
+  # --------------------------------
+  # The clusterXXX information is used to setup the cluster for
+  # use by ProxySQL.
+  #
+  clusterHost="localhost"
+  clusterPort=4110
+  clusterUser="admin"
+  clusterUserPassword="admin"
+
+  # --------------------------------
+  # ProxySQL will use SSL to connect to the backend servers
+  #
+  useSSL=0
+
+  # --------------------------------
+  # Max number of connections from ProxySQL to the backend servers.
+  #
+  maxConnections=1000
+
+  # Sets the proxysql's mysql-monitor_galera_healthcheck_interval variable
```

Comment:
This is wrong. This value given here is passed on proxysql's `interval_ms` field in the `scheduler` table.
In simple words, this value is used by the percona-scheduler-admin script and causes the pxc_scheduler_handler to be run every `nodeCheckInterval` milliseconds.

Reply:
This information is taken from the example configuration file in https://github.com/percona/proxysql-admin-tool
Fix that please.

Reply:
We have fixed it in the [release branch](https://github.com/percona/proxysql-admin-tool/tree/v2.3.2-1-dev). So, it is not visible in the default(v2) branch.

Reply:
Copied latest version